orchestrator/api/system.py

```
@@ -335,10 +335,143 @@ async def get_system_health(db: Session = Depends(get_db)):
         
         # Check database connection
         db_status = "healthy"
+        db_metrics = {}
         try:
             db.execute("SELECT 1")
-        except Exception:
+            db_metrics = {"connection": "active"}
+        except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)